ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/tests/PackageManagement.scala

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) 2019 The DAML Authors. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.api.testtool.tests
+
+import com.daml.ledger.api.testtool.infrastructure.{LedgerSession, LedgerTest, LedgerTestSuite}
+import com.digitalasset.ledger.packagemanagementtest.PackageManagementTest.PackageManagementTestTemplate
+import com.digitalasset.ledger.packagemanagementtest.PackageManagementTest.PackageManagementTestTemplate._
+import com.google.protobuf.ByteString
+import io.grpc.Status
+
+import scala.concurrent.{ExecutionContext, Future}
+
+final class PackageManagement(session: LedgerSession) extends LedgerTestSuite(session) {
+
+  private[this] val testPackageResourcePath =
+    "/ledger/ledger-api-test-tool/PackageManagementTest.dar"
```

Comment:
Why do we need a new dedicated DAR file for this test?

Reply:
It would probably be interesting to have a chance to upload something for the first time during this test. No guarantee, but uploading `Test.dar` guarantees the opposite, which is probably not desirable.

Reply:
We cannot make this guarantee, unless we synthesize a dalf package on the fly (which is likely not impossible). Anything else would be making assumptions during the test run that we cannot make (e.g. that the test tool in a specific release has never been run before against the same ledger).